tasks/db_deploy/install/create_db.py

```diff
@@ -41,10 +40,29 @@ def create_fresh_orca_install(config: Dict[str, str]) -> None:
         create_recovery_objects(conn)
         create_inventory_objects(conn)
 
+        # Create internal reconciliation objects
+        create_internal_reconciliation_objects(conn)
```

Comment:
should `create_internal_reconciliation_objects(conn)` be inside `create_fresh_orca_install()`?